src/coreclr/src/jit/CMakeLists.txt

```diff
@@ -322,7 +322,7 @@ else()
   elseif(CLR_CMAKE_HOST_SUNOS)
```

Comment:
It is interesting that we test for target everywhere except one case - SunOS where we check for host.

Reply:
@am11 Is there a reason for this?